rX/snapshot-generation/nl-core-NursingIntervention-output.json

```diff
@@ -2318,7 +2300,6 @@
         }],
         "code" : "http://hl7.org/fhirpath/System.String"
       }],
-      "fixedUri" : "http://nictiz.nl/fhir/StructureDefinition/ext-CarePlan-MaterialUsed",
```

Comment:
Not sure if this is due to the fact that the extension is not included in the test case, but I think it should not be removed?